wagtail/admin/ui/side_panels.py

```diff
@@ -155,3 +149,18 @@ def media(self):
         for panel in self.side_panels:
             media += panel.media
         return media
+
+
+class PageSidePanels(BaseSidePanels):
```

Comment:
Does it make sense to move this into the view itself? To me, it feels like the logic that decides which UI components to render should be separate from the components themselves

Reply:
I think so. Do you mean something like, keeping the `BaseSidePanels` (maybe rename it to `SidePanels`) but make the `side_panels` a parameter to the `__init__`?

Reply:
It's used in the create, edit, and revert revisions views though, so removing this class would mean we have to replicate the logic across the three files. Not saying this is something that should be avoided, just thought it might be worth considering.